src/lib/Scenes/Artwork/Components/PartnerCard.tsx

```diff
@@ -0,0 +1,147 @@
+          onCompleted: () => this.handleShowSuccessfullyUpdated(),
+          mutation: graphql`
+            mutation PartnerCardMutation($input: FollowProfileInput!) {
+              followProfile(input: $input) {
+                profile {
+                  gravityID
+                  internalID
+                  is_followed
+                }
+              }
+            }
+          `,
+          variables: {
+            input: {
+              profile_id: profileID,
+              unfollow: partnerFollowed,
+            },
+          },
+          optimisticResponse: {
+            followProfile: {
+              profile: {
+                internalID: profileID,
+                gravityID: partnerSlug,
+                is_followed: !partnerFollowed,
+              },
+            },
+          },
+        })
+      }
+    )
+  }
+
+  handleTap(href: string) {
+    SwitchBoard.presentNavigationViewController(this, href)
+  }
+
+  handleShowSuccessfullyUpdated() {
+    this.setState({
+      isFollowedChanging: false,
+    })
+  }
+
+  render() {
+    const { artwork } = this.props
+    const partner = artwork.partner
+    const galleryOrBenefitAuction = artwork.sale && (artwork.sale.isBenefit || artwork.sale.isGalleryAuction)
+    if (partner.type === "Auction House" || galleryOrBenefitAuction) {
+      return null
+    }
+    const { isFollowedChanging } = this.state
+    const imageUrl = partner.profile ? partner.profile.icon.url : null
+    const locationNames = get(partner, p => limitWithCount(filterLocations(p.locations), 2), []).join(", ")
+
+    return (
+      <Flex>
+        <TouchableWithoutFeedback onPress={this.handleTap.bind(this, partner.href)}>
+          <EntityHeader
+            name={partner.name}
+            href={partner.is_default_profile_public && `${partner.href}`}
+            meta={locationNames}
+            imageUrl={imageUrl}
+            initials={partner.initials}
+            FollowButton={
+              partner.profile && (
+                <InvertedButton
+                  grayBorder={true}
+                  text={partner.profile.is_followed ? "Following" : "Follow"}
+                  onPress={this.handleFollowPartner.bind(this)}
+                  selected={partner.profile.is_followed}
+                  inProgress={isFollowedChanging}
+                />
+              )
+            }
+          />
+        </TouchableWithoutFeedback>
+      </Flex>
+    )
+  }
+}
+
+export const PartnerCardFragmentContainer = createFragmentContainer(PartnerCard, {
+  artwork: graphql`
+    fragment PartnerCard_artwork on Artwork {
+      sale {
+        isBenefit
+        isGalleryAuction
+      }
+      partner {
+        is_default_profile_public
+        type
+        name
+        gravityID
+        internalID
+        id
+        href
+        initials
+        profile {
+          internalID
+          gravityID
+          is_followed
+          icon {
+            url(version: "square140")
```

Comment:
Don’t request specific resolution URLs, the `OpaqueImage` component handles that based on the actual layout and screen density. Instead simply request the url without args and make sure your image component is laid out correctly.

Reply:
For some reason when I remove the resolution it breaks the circular crop of the image. I'm specifying a width and a height and it appears the same size its just not circular. Do you know why this might be happening?

Reply:
Unsure off-hand, as I have no experience with that component. Perhaps @l2succes does?

Reply:
I took a quick look at the `Avatar` component with Lily yesterday, despite the fact that it's not using `OpaqueImage` I could find an obvious reason why this is happening. My suggestion was to track down where the `corner-radius` css prop was coming from first.

Reply:
Ok, if no solution can be found right away, can you file a ticket @l2succes with the context you have? As this is tech debt that will proliferate and definitely not something we should be doing.